pkg/advisory/store.go

```diff
@@ -54,3 +54,20 @@ type Putter interface {
 	// returned.
 	Upsert(ctx context.Context, request Request) (string, error)
 }
+
+// MapByVulnID maps the given advisories by their vulnerability ID, creating a
```

Comment:
This seems like an implementation detail that does not belong in store.go (which I think should be used for the interface definition).

Reply:
That sounds good, I can move it somewhere else

Reply:
Moved to a new file